src/model/profile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const profileSchema = mongoose.Schema({
```

Comment:
Won't dock points since the lab didn't explicitly say it, but try not to use the same schemas demo'd in class.